hil-test/tests/spi_half_duplex_write_psram.rs

```diff
@@ -0,0 +1,194 @@
+//! SPI Half Duplex Write Test
+//% FEATURES: psram
+//% CHIPS: esp32s3
```

Comment:
I think I would prefer merging with with the other tests. You can add conditional cfgs around the PSRAM setup and for the PSRAM test, but I think you can reuse the same test logic for this.

Reply:
I'll handle that when its closer to done

Reply:
So how can I limit the psram feature this needs to esp32s3 only?

Reply:
Might be best to change this to a PSRAM test rather than an spi test. In terms of the test file name I mean

Reply:
Good question, I guess you currently can't. We can introduce `FEATURES(esp32s3): ` later to do this, but I guess it's okay to have the test suite separate unitl then.

Reply:
> Might be best to change this to a PSRAM test rather than an spi test. In terms of the test file name I mean

Maybe, but my issue was that this essentially duplicates test logic, which is just not ideal in terms of maintainability. We'll have to improve so that we can avoid the need to do it.

Reply:
After https://github.com/esp-rs/esp-hal/pull/2175 you should be able to add `//% CHIP-FEATURES(esp32s3): psram`